scss/_settings_grid.scss

```
@@ -5,15 +5,29 @@
 $grid-max-width: 80rem !default;
 $grid-columns-small: 4 !default;
 $grid-columns-medium: 6 !default;
+$grid-columns-medium-new: 4 !default;
 $grid-columns: 12 !default;
+
 $threshold-4-6-col: $breakpoint-small !default;
+$threshold-4-small-4-med-col: $breakpoint-small !default;
 $threshold-6-12-col: $breakpoint-large !default;
+$threshold-4-med-12-col: $breakpoint-large !default;
 
 $grid-column-prefix: 'col-' !default;
+$grid-row-class: 'row' !default;
+$grid-row-4-cols-class: '#{$grid-row-class}--4-cols' !default;
 $grid-small-col-prefix: '#{$grid-column-prefix}small-' !default;
 $grid-medium-col-prefix: '#{$grid-column-prefix}medium-' !default;
 $grid-large-col-prefix: #{$grid-column-prefix} !default;
 
+// Common grid patterns
+$col-50-medium: calc($grid-columns-medium / 2);
+$col-50-medium-new: calc($grid-columns-medium-new / 2);
```

Not sure if we need to complicate this. We know exactly how many columns is there.

```suggestion
$col-50-medium-4-cols: calc(4 / 2);
```

Maybe even directly say 
```suggestion
$col-50-medium-4-cols: 2;
```

---

Removed this variable as it is not needed currently.